woocommerce/woocommerce-admin
# Add remote inbox notifications rule that triggers when WCA is updated

Fixes #5957

#5957 wants a message to be displayed after upgrading the WooCommerce Admin version (actually #5957 wants to trigger on the WooCommerce update, but these are practically the same).

This adds a hook to the WCA upgrade process (action hook `woocommerce_admin_updated`) which sets a flag in an option, goes through the Remote Inbox Notifications run, then clears the flag. It also adds a new `wca_updated` rule which just checks on that flag being set.

### Detailed test instructions:

1. add the test URL to the `DATA_SOURCES` array in `DataSourcePoller.php`: `https://gist.githubusercontent.com/becdetat/4fda7403cd34f49b71bccd147419e34e/raw/7360ec1f0dad9ecc867c62a73dad4006635c2b84/wca-updated.json`
2. Run the `wc_admin_daily` cron  job, which will load in the new spec
3. Open the WooCommerce dashboard. The new note should _not_ appear.
4. Trigger a WCA upgrade by bumping the version option down in the database: `UPDATE wp_options SET option_value = '1.8.0' WHERE option_name = 'woocommerce_admin_version'`
5. Refresh the dashboard. The new note should now be present.
